src/client/vendor/SanitizeConfig.js

```diff
@@ -160,9 +160,11 @@ export default ({ large = true, noImage = false, sanitizeErrors = [], secureLink
       if (!href) href = '#';
       href = href.trim();
       const attys = {};
-      // If it's not a (relative or absolute) steemit URL...
-      if (secureLinks && !href.match(/^^(\/|https:\/\/(staging\.)?busy\.org(?![\w\.]+))/)) {
+      if (!href.match(/^^(\/|https:\/\/(staging\.)?busy\.org(?![\w\.]+))/)) {
         attys.target = '_blank';
+      }
+      // If it's not a (relative or absolute) steemit URL...
```

Comment:
This comment belonged to different condition (regex match). I don't think it worth keeping this comment at all.